test/latexify.jl

```diff
@@ -1,111 +0,0 @@
-#! format: off
```

Comment:
Not sure, this does not show up on my computer? For some reason the main latexify file keeps getting changed to delted/created instead of moved, are you talking about the old latexify test file?